src/extension.ts

```diff
@@ -8,12 +8,16 @@ import { AwsContextCommands } from "./aws_context";
 import { DefaultEMRClient } from "./clients/emrClient";
 import { DefaultEMRContainersClient } from "./clients/emrContainersClient";
 import { DefaultEMRServerlessClient } from "./clients/emrServerlessClient";
+import { DefaultGlueClient } from "./clients/glueClient";
 import { EMREC2Filter } from "./emr_explorer";
 import { EMRLocalEnvironment } from "./emr_local";
 import { copyIdCommand } from "./explorer/commands";
 import { EMRContainersNode } from "./explorer/emrContainers";
 import { EMRNode } from "./explorer/emrEC2";
 import { EMRServerlessNode } from "./explorer/emrServerless";
+import { GlueCatalogNode } from "./explorer/glueCatalog";
+import { getWebviewContent, GlueTablePanel } from "./panels/glueTablePanel";
```

Comment:
I don't think `GlueTablePanel` is used.
```suggestion
import { getWebviewContent } from "./panels/glueTablePanel";
```

Reply:
Agree, that's left from one of my experiments with web views.